src/Ushahidi/Modules/V5/Actions/Role/Commands/CreateRolePermissionCommand.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Ushahidi\Modules\V5\Actions\Role\Commands;
+
+use App\Bus\Command\Command;
+
+class CreateRolePermissionCommand implements Command
+{
+    /**
+     * @var string
+     */
+    private $role;
+
+    /**
+     * @var string
+     */
+    private $permission;
+   
+
+    public function __construct(string $role, string $permission)
+    {
+        $this->setRole($role);
```

Comment:
These setters are redundant. We don't want to have public setter methods since we don't want Commands to change its state.